### PR TITLE
Fix some spelling errors in docs

### PR DESCRIPTION
**Pull request by Gama11**
You have vscode-spell-checker to thank for this.

### CODE REVIEW

File: std/haxe/macro/Expr.hx [hunk 1]
```diff
@@ -67,7 +67,7 @@ enum Constant {
 	CString( s : String );
 
 	/**
-		Represents an indentifier.
+		Represents an identifier.
```
Comment (Gama11) [inline std/haxe/macro/Expr.hx]: I wonder what an `indentifier` would look like.

**Comment by Simn** [comment]
This PR makes me feel more incompetent than I care to admit...

**Comment by Gama11** [comment]
I was quite surprised by the number I found too, I thought the std lib was pretty spelling-error-free overall.

**Comment by Gama11** [comment]
@Simn This might make you feel a little better: https://github.com/HaxeFlixel/flixel/commit/d5f1382c48925d35d3ecb58e01518913f9162098 x)

**Comment by Simn** [comment]
It does, thanks!

**Pull request merged**